modules/containers-signature-verify-unsigned.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-container-signature.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="containers-signature-verify-artifacts_{context}"]
+= Understanding the verification of container images lacking verifiable signatures
+Each {product-title} release image is immutable and signed with a Red Hat production key. During an {product-title} update or install, a release image might deploy container images that do not have verifiable signatures. Each signed release image digest is immutable. The references in the signed release image digest are to the immutable digest of another image, so the contents can be trusted transitively. In other words, the signature on the release image validates all release contents.
```

Comment:
this may seem slightly pedantic but... the _release image_ contains the references; the immutable _digest_ of the release image is what is signed. the digest itself doesn't reference anything.
```suggestion
Each {product-title} release image is immutable and signed with a Red Hat production key. During an {product-title} update or install, a release image might deploy container images that do not have verifiable signatures. Each signed release image digest is immutable. Each reference in the release image is to the immutable digest of another image, so the contents can be trusted transitively. In other words, the signature on the release image validates all release contents.
```